Update tls-ssl.md (#2681)
RedisLabs/redislabs-docs: content/rc/security/database-security/tls-ssl.md

- Publicly trusted GlobalSign Root CA

{{<note>}}

> The downloaded PEM file contains multiple certificates, so please make sure to import **all** certificates to your client trust store. Client code often imports the first certficate only unless done properly.

Reviewer: ```suggestion
The downloaded PEM file contains multiple certificates. Make sure to import **all** certificates to your client trust store. Some clients will only import the first certificate by default.
```

Author: Thanks @cmilesb! One request: `Client code often imports` refers to how users implement the code that imports the certificates and not necessarily to how clients read them, so that is an important distinction. I'll leave it up to you to write that better.

Author: Sounds good!
